sdk/core/Azure.Core/src/Serialization/DateTimeHandling.cs

```diff
@@ -10,9 +10,9 @@ public enum DateTimeHandling
     {
         /// <summary>
         /// DateTime and DateTimeOffset values will be serialized and deserialized using
-        /// System.Text.Json defaults.
+        /// System.Text.Json default ISO 8601-1:2019 format.
         /// </summary>
-        None,
+        Iso8601,
```

Comment:
Curious. For all intents and purposes these are the same. RFC3339 actually supersedes ISO8601, which is a proper subset of RFC3339 now. Is there a specific difference you're after? Was this related to the conversation about the TZ offsets on another PR? Might be good to add to the doc comments what the specific difference is these represent. As a developer, I wouldn't be sure which to use or why.

Reply:
What I was trying to convey with this is that we won't pass anything to the STJ JsonSerializer if you pass this setting, but that the effect will be you get [their default DateTime serialization format](https://learn.microsoft.com/en-us/dotnet/standard/datetime/system-text-json-support#support-for-the-iso-8601-12019-format).  I wanted it to be clear what that format is, hence the name change.  Do you think the updated comment reflects that?  Is your suggestion to update the doc comments so it's clear how our implementation of the RFC3339 JsonConverter differs from the STJ default?

Reply:
It wasn't entirely clear until you mentioned that. It reads well enough, but I think all the discussions we've had on this might've added to my confusion. Probably fine as-is.

Reply:
Any confusion is good feedback - I'll punch it up to clarify a bit.  Thank you!